.dockerignore

```diff
@@ -13,6 +13,3 @@
 **/.idea
 **/.kmigrator
 **/docs
-**/.config
-**/.yarn
```

Comment:
Why you don't want to ignore these cache dirs? Is that for speed up build time? May this change cause bugs related to cache?

Reply:
We already have .yarn inside out repository folder. We can't ignore them